src/DataObject/Controller/TreeController.php

```diff
@@ -78,7 +78,7 @@ public function __construct(
     #[PageSizeParameter]
     #[ParentIdParameter(
         description: 'Filter data objects by parent id.',
-        example: null
+        example: 12
```

Comment:
```suggestion
        example: 1
```
Let's rather use 1 here since it would be the root folder.